packages/api/src/controllers/signing-key.ts

```diff
@@ -241,4 +242,48 @@ signingKeyApp.patch(
   }
 );
 
+signingKeyApp.get(
+  "/jwt/:playbackId",
+  authorizer({ anyAdmin: true, noApiToken: true }),
```

Comment:
You should just use `admin:` (not `anyAdmin:`) instead which is the general standard on these dashboard only apis (like /user APIs I think). Don't remember what's the difference with `noApiToken` tho 🤔 
```suggestion
  authorizer({ admin: true }),
```